packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -192,11 +193,25 @@ const scriptReplacer = (
 export type BootstrapScriptDescriptor = {
   src: string,
   integrity?: string,
+  crossOrigin?: string,
 };
 export type ExternalRuntimeScript = {
   src: string,
   chunks: Array<Chunk | PrecomputedChunk>,
 };
+
+/**
+ * We normalize crossOrigin to be "use-credentials", "" or null.
+ * Unless using "use-credentials" all the other strings behave like "anonymous" and "" is
+ * shortest bytes-wise.
+ */
+function getCrossOrigin(options?: mixed) {
+  return options == null || typeof options.crossOrigin !== 'string'
```

Comment:
I think overall nested ternary expressions require more mental overhead to reason about or to edit in the future..

```typescript
if (options == null || typeof options.crossOrigin !== 'string') {
  return null;
}
if (options.crossOrigin === 'use-credentials') {
  return 'use-credentials';
}
return '';
```